cmd/operator-sdk/bundle/create_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/bundle"
+)
+
+var _ = Describe("Running a bundle validate command", func() {
+	Describe("NewCreateCmd", func() {
```

Comment:
```suggestion
	Describe("NewCreateCmd should", func() {
```

Reply:
I prefer to leave those describes as just the name of the method they're testing